src/main/java/org/folio/linked/data/model/entity/Resource.java

```diff
@@ -193,6 +226,9 @@ void prePersist() {
       throw new IllegalStateException("Cannot save resource [" + id + "] with types " + types + ". "
         + "Folio metadata can be set only for instance and authority resources");
     }
+    if (isNull(this.createdDate)) {
+      this.createdDate = new Timestamp(System.currentTimeMillis());
```

Comment:
now we don't need this code do we?
aren't we setting the ```createdDate``` using ```touchForCreate``` method?

Reply:
not in all cases, we need it for the case with resource update